thread-rcu/Makefile

```diff
@@ -0,0 +1,23 @@
+CFLAGS = -Wall
+CFLAGS += -g
+CFLAGS += -std=c11
+CFLAGS += -D'N_READERS=10'
+CFLAGS += -fsanitize=thread
+LDFLAGS += -lpthread
+
+all:
```

Comment:
You can rewrite as following for modularization:
```
main: main.c
        $(CC) -o $@ $< $(CFLAGS) $(LDFLAGS)

clang: CC=clang
clang: main
                                                                                                                                                              
all: main
```

Then, `make` will trigger the default compiler, and `make clang` invokes `clang` explicitly.